packages/builder/package.json

```diff
@@ -53,7 +53,7 @@
     "@budibase/shared-core": "*",
     "@budibase/string-templates": "*",
     "@budibase/types": "*",
-    "@codemirror/autocomplete": "^6.7.1",
+    "@codemirror/autocomplete": "6.9.0",
```

Comment:
Fixing this, as while trying to get newer ones I got conflicts with the other packages